apps/client/src/features/rundown/Rundown.tsx

```diff
@@ -104,7 +104,7 @@ export default function Rundown(props: RundownProps) {
               return;
             }
             // eslint-disable-next-line @typescript-eslint/no-non-null-assertion -- we check for this before
```

Comment:
Is there a null assertion happening on the next line?

Reply:
yeah.... good catch, dont know why this was there
🤷‍♂️ 

I actually went down in trying to get eslint to mark an error on unused directive, but this would have meant adding a new plugin, which I fear would add more weight to maintenance than benefit

Either way, good catch